integration-tests/contract-fetch-multiple-big-map-keys.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { CONFIGS } from './config';
+import { tokenCode } from './data/tokens';
+import { MichelsonMap, BigMapAbstraction } from '@taquito/taquito';
+import BigNumber from 'bignumber.js';
+
+CONFIGS().forEach(({ lib, rpc, setup }) => {
+    const Tezos = lib;
+    const test = require('jest-retries');
+
+    describe(`Test contract origination with multiple BigMap variations through contract api using: ${rpc}`, () => {
+        /**  The contract code doesn't have annotations, so the keys of the storage and of the bigmap are indexes. */
+
+        beforeEach(async (done) => {
+            await setup();
+            done();
+        });
+
+        test('Verify contract.originate for a contract with a BigMap with multiple values to be indexed (also fetching the Storage/BigMap)', 2, async (done: () => void) => {
```

Comment:
jest-retries and `done: () => void` can be removed

Reply:
removed